### ISSUE TITLE
Render external view

### ISSUE THREAD

**Issue by mridah** [slimphp/Slim#3179]
Suppose I have an external url `example.com/aaaa`, I want to have a route that would forward all requests to that domain and return the results ( **NOTE: I do not want a redirect, I don't want the URL to change** )

I tried this but it doesn't work:

    $app->get('/mailers/{route}', function( $request, $response, $args) {
        $app->render('http://example.com/' . $args['route'] . '.html');
    });

**_So basically both `my-domain/aaaa` and `my-other-domain/aaaa` should return the same page._**

Is there a work around?

**Comment by RyanNerd** [comment]
Maybe using a [custom view](https://www.slimframework.com/docs/v2/view/custom.html) and in the render method return  `get_file_contents('http://example.com/xyz');`

**Comment by l0gicgate** [comment]
I'm closing this as resolved
